.github/workflows/critical-issue.yml

```diff
@@ -1,17 +1,38 @@
 name: Slack notification for critical issue
 on:
   issues:
+  pull_request:
```

Comment:
:x: this would make it react on all issue events (before it was "labeled" only).

Reply:
Yeah, I know. But in this way I can easily test my changes 👍